federation/manifests/federation-controller-manager-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{.FEDERATION_CONTROLLER_MANAGER_DEPLOYMENT_NAME}}
+  namespace: {{.FEDERATION_NAMESPACE}}
+  labels:
+    app: federated-cluster
+spec:
+  template:
+    metadata:
+      name: federation-controller-manager
+      labels:
+        app: federated-cluster
+        module: federation-controller-manager
+    spec:
+      containers:
+      - name: controller-manager
+        image: {{.FEDERATION_CONTROLLER_MANAGER_IMAGE_REPO}}:{{.FEDERATION_CONTROLLER_MANAGER_IMAGE_TAG}}
+        command:
+        - /usr/local/bin/federation-controller-manager
+        - --master={{.FEDERATION_API_HOST}}:{{.FEDERATION_API_PORT}}
```

Comment:
This is the publicly exposed endpoint, either via nodeport or lb depending on provider. Instead you should refer here to the federated-apiserver service by it's cluster DNS name: 

`--master=https://{{.FEDERATION_APISERVER_DEPLOYMENT_NAME}}:443`